docs/docs/recipes/user-profile/README.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 7
+---
+
+# 👤 User Profiles
+
+In previous sections, we demonstrated how to manage your users [using Admin Console](/docs/recipes/manage-users/admin-console), or through [management APIs](/docs/recipes/manage-users/management-api). This is a straightforward process for those in the role of administrator. However, it is also important for the regular users to update their profiles on their own.
```

Comment:
Not sure about the user context when reading this article. Could we avoid mentioning  `In previous sections`?

Reply:
I think it's OK since the "previous sections" means the first several recipes in the same folder. User profile is currently the last one in all recipes, not to mention I have also provided the links to the "previous sessions" I referred to.